app/models/academic_cycle.rb

```diff
@@ -4,6 +4,14 @@ class AcademicCycle < ApplicationRecord
   validates :start_date, :end_date, presence: true
   validate :start_date_before_end_date
 
+  scope :trainees_filter, -> { where("start_date >= ?", "2020-09-01".to_date).order(:start_date) }
```

Comment:
Not sure if it's worth doing in this PR but it would be good to make this more dynamic. Assuming we always want to show 3 cycles in the filter: (current, last and next) the values for these will always change with each cycle

Reply:
@darokel 
I thought we wanted only to see from 2020. But I'll change it so it shows last 3 cycles.

Reply:
@edwardhorsford could you confirm? Might as well product review this too